docs/source/features/metrics.md

```diff
@@ -50,39 +50,40 @@ server.listen().then(({ url }) => {
 You can provide your Graph Manager API key to Apollo Server via the `ENGINE_API_KEY` environment variable. Similarly, you can assign a particular [variant](https://www.apollographql.com/docs/platform/schema-registry/#managing-environments)
 to an Apollo Server instance via the `ENGINE_SCHEMA_TAG` environment variable.
 
-You can set environment variable values on the command line as seen below, or by using the [`dotenv` npm package](https://www.npmjs.com/package/dotenv) (or similar).
+You can set environment variable values on the command line as seen below, or with the [`dotenv` npm package](https://www.npmjs.com/package/dotenv) (or similar).
 
 ```bash
 # Replace the example values below with values specific to your use case.
 ENGINE_API_KEY=YOUR_API_KEY ENGINE_SCHEMA_TAG=development node start-server.js
 ```
 
-### Client awareness
+### Identifying distinct client versions
```

Comment:
For my own curiosity and understanding: Is the idea here to change the title to describe the user action rather than the feature?

Reply:
Correct. This isn't an across-the-board change that we need to make or anything, but since "client awareness" isn't a universally known term, it felt appropriate to err on the side of the task instead of the feature in this case.